framework/console/Controller.php

```diff
@@ -254,6 +254,19 @@ public function stderr($string)
      *  - validator: a callable function to validate input. The function must accept two parameters:
      *      - $input: the user input to validate
      *      - $error: the error value passed by reference if validation failed.
+     * 
+     * An example of how to use the prompt method with a validator function.
+     * 
+     * ```php
+     * $code = $this->prompt('Enter 4-Chars-Pin', ['required' => true, 'validator' => function($input, &$error) {
+     *     if (strlen($input) !== 4) {
+     *         $error = 'The Pin must be at least 4 chars!';
```

Comment:
`at least 4 chars` means 4 chars or more, but the check is whether it is equal to 4. should be changed.